tests/logger.rs

```diff
@@ -26,7 +25,7 @@ mod test_module {
                 payload: String,
             ) -> FFIResult<()> {
                 assert!(matches!(log_level, telio::ffi_types::TelioLogLevel::Info));
-                assert_eq!(r#""logger::test_module":43 test message"#, payload);
+                assert_eq!(r#""logger::test_module":42 test message"#, payload);
```

Comment:
Where does this number comes from? Maybe some named const for it ..?

Reply:
So this is the line of info!() trace at the top, for my test I made it relative, i will make it smarter here also

Reply:
Ok so turning this is no an easy task here, how currently tests are design, this will have to be left as is. Once cell would i think just add additional confusion

Reply:
Maybe making some const with that number can be considered, so it will be more apparent?

Reply:
added a const